script.js

```diff
@@ -1,4 +1,89 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (
+    input == "scissors" ||
+    input == "stone" ||
+    input == "paper" ||
+    input == "Scissors" ||
+    input == "Stone" ||
+    input == "Paper"
```

Comment:
Good job accounting for both inputs with capitalization and without! If you want to avoid this all together, you can use something like `input.toLowerCase() == "scissors" ...` to force everything to be lowercase.

https://www.w3schools.com/jsref/jsref_toLowerCase.asp